drivers/ieee802154/ieee802154_bm_serial.c

```diff
@@ -338,16 +338,22 @@ static int ieee802154_bm_serial_tx(const struct device *dev,
 	{
 		return -EIO;
 	}
+	
+	bm_frame_header_t bm_frm_hdr = {.version= BM_V0, .payload_type= BM_IEEE802154, .payload_length= len + 2, .header_crc= 0}; // account for the CRC16
```

Comment:
Not important to change now, but this CRC could probably be a simple CRC8, since the likelihood of occurrence, false positive rate, and impact of false positive all combine to be fairly low. Saves us a two bytes of manchester coded overhead.

Reply:
I wonder if we even need a header CRC. Like you say, the likelihood of occurrence and impact is low. If having a single CRC at the end of the frame is good enough for ethernet, it's probably good enough for us 😅

I worry about using various different kinds of CRCs around the code. It's going to be confusing to anyone else trying to understand and implement.